tapir/coop/tests/test_create_membershipresignation.py

```diff
@@ -15,27 +16,17 @@ def test_membershipResignation_created_isTrue(self):
         resigned_member = MembershipResignationFactory.create()
```

Comment:
If you use `create`, an object with be created in the database.
Since you want to call the create view, you just want to create a python resignation object but not put it in the database already. This is done with `build` instead of `create`.